### PR TITLE
 Move “Resource Bin Packing” into Scheduling & Eviction section

### LINKED ISSUE #22534: Move “Resource Bin Packing” into Scheduling & Eviction section
**This is a Feature Request**

**What would you like to be added**
Migrate the content from https://kubernetes.io/docs/concepts/configuration/resource-bin-packing/ into somewhere inside https://kubernetes.io/docs/concepts/scheduling-eviction/

**Why is this needed**
https://kubernetes.io/docs/concepts/configuration/resource-bin-packing/ is not about configuration in the way that ConfigMap and Secret are; instead, it's much more about scheduling.

**Comments**
/language en
/kind cleanup

Initially, a straightforward cut-and-paste would be fine, so I'll mark this as:
/good-first-issue

### PR DESCRIPTION
**Pull request by sftim**
This PR combines changes from PRs #23359 and #23390 (work done by @c-mcdonnell)

Fixes #22534
/kind cleanup
/sig scheduling

### CODE REVIEW

File: static/_redirects [hunk 1]
```diff
@@ -497,3 +497,4 @@
 /docs/setup/cluster-large/     /docs/setup/best-practices/cluster-large/   301
 /docs/setup/node-conformance/     /docs/setup/best-practices/node-conformance/   301
 /docs/setup/certificates/     /docs/setup/best-practices/certificates/   301
+/concepts/configuration/resource-bin-packages/   /concepts/scheduling-eviction/resource-bin-packages/ 301
```
Comment (sftim) [inline static/_redirects:500]:
```suggestion
/concepts/configuration/resource-bin-packing/   /concepts/scheduling-eviction/resource-bin-packing/ 301
```

Comment (bradtopol) [reply static/_redirects:500]: /assign
hi @sftim does this need a technical review or is this just moving things around?  Let me know and I will line up someone from SIG scheduling to do a technical review. Thanks

Comment (sftim) [reply static/_redirects:500]: This is just a rename.

**Comment by sftim** [comment]
Done in https://github.com/kubernetes/website/pull/23529

/close

**Pull request closed without merging**